aries-site/src/examples/components/tag/data.js

```diff
@@ -0,0 +1,32 @@
+const departments = [
```

Comment:
Because tags can really be anything, I think a more realistic data structure returned from an API might resemble something more like:

```
{
    tags: [
        Department: [
            'Finance',
            'Legal',
            'Engineering',
            'Design',
            'Human Resources',
        ],
        Environment: [ ... ],
        'Time Zone': [ ... ],
        Location: [ ... ],
        'favorite',
        'bug',
    ],
}
```

Then when consuming the data we'd check the type of the tags --> IF typeof === 'object', THEN we know we have a name-value tag; IF typeof === 'string', THEN we know we have a simple, plain tag.

Reply:
Sounds good, for this example I'll stick to just the name-value pairs and we can create a separate simplified example of value-only.